src/igvfd/schemas/changelogs/prediction_set.md

```diff
@@ -3,6 +3,7 @@
 ### Schema version 3
 
 * Disallow empty strings in `description`.
+* Allow underscores in the pattern for `chromosome` in `loci`.
```

Comment:
Minor comment, looks like the property in prediction set is specifically "targeted_loci"

Reply:
fixed